applications/ParticleMechanicsApplication/custom_utilities/mpm_particle_generator_utility.cpp

```diff
@@ -7,390 +7,31 @@
 //  License:		 BSD License
 //					 Kratos default license: kratos/license.txt
 //
-//  Main authors:    Ilaria Iaconeta, Bodhinanda Chandra
+//  Main authors:    Bodhinanda Chandra
```

Comment:
Haha

Reply:
I don't mean to do that hahaha I deleted the MPM_strategy and created a new mpm_particle_generator_utility